_includes/sidebar.html

```diff
@@ -43,6 +42,49 @@ <h3>Getting Started</h3>
         <li><a href="https://github.com/travis-ci/travis#readme">Command Line Client</a></li>
       </ul>
 
+      <h3>Customizing your Build</h3>
+      <ul>
+
+<li><a href="#Installing-a-Second-Programming-language">Installing a Second Programming language</a></li>
+<li><a href="#Build-Timeouts">Build Timeouts</a></li>
+<li><a href="#Limiting-Concurrent-Jobs">Limiting Concurrent Jobs</a></li>
+<li><a href="/user/speeding-up-the-build/">Speeding up the Build</a></li>
+
+
+<!-- group these? -->
+<li><a href="#Building-only-the-latest-commit">Building only the latest commit</a></li>
```

Comment:
I think Building only the latest commit groups more with speeding things up and Travis config details, above - rather than git config related. What do you think? Given that mind-mapping depends on folks, I'm super curious to hear your thoughts :)

Reply:
:100: the Git is 'fake' grouping really - as in it is a bucket where people will find stuff. You're unlikely to look for git commit depth, say, and also go OH, here is another git thing I'm interested in